Dockerfile

```diff
@@ -1,72 +1,22 @@
-# EXT_REGISTRY allows specifying a different registry that is acting as a pull-through cache for Docker Hub.
-# To provide EXT_REGISTRY at build time, supply it as a build argument
-#     `docker build . --build-arg EXT_REGISTRY=https://myregistry.com/`
-# Note that the value MUST contain a trailing slash
-# If not provided, images will be pulled from Docker Hub
-ARG EXT_REGISTRY=""
-ARG IMAGE_TAG="11.4"
+FROM python:3.10-slim-bullseye
 
-FROM ${EXT_REGISTRY}docker.io/library/debian:${IMAGE_TAG} as proj
-# Renew build args for the stage
-ARG EXT_REGISTRY IMAGE_TAG
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y wget git libopenblas-dev imagemagick g++ make imagemagick
 
-ARG PROJ_VERSION='9.0.1'
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository -y ppa:ubuntugis/ppa
+RUN apt-get install -y gdal-bin libgdal-dev
+RUN pip install -U pip 
 
-WORKDIR /build
+# could install the rest of geoips dependancies here
+RUN pip install rasterio
```

Comment:
Does this need to be installed first? It does not successfully build during the geoips installation?  It should be included as a dependency in geoips?

Reply:
It doesn't need to be installed first, but if the end goal is a super-small image I'm thinking it'd be advantageous to build geoips dependencies and then pull them over to a new image without their build tooling

Reply:
Ahh, so this is more of a proof of concept / start of a multistage build.  Got it.